src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
 		// Find delegate handlers
 		// Black-hole SVG <use> instance trees (#13180)
+		//
+		// Support: Firefox
```

Comment:
The Firefox ticket https://bugzilla.mozilla.org/show_bug.cgi?id=184051 for this is still open, so either my test case sucks or this is a dup of a ticket that was fixed.